doc/multi-o.md

```diff
@@ -0,0 +1,123 @@
+
+More detailed instructions for connecting to Ethereum can be found [here](https://livepeer.readthedocs.io/en/latest/quickstart.html#connecting-to-an-ethereum-node). 
+
+```
+livepeer \
+    -network <mainnet|rinkeby> \
+    -ethUrl <JSON_RPC_PROVIDER> \
+    -ethKeystorePath <PATH_TO_KEYSTORE_DIR> \
+    -ethAcctAddr <ETHEREUM_ADDRESS> \
+    -ethPassword <ETHEREUM_ACCOUNT_PASSWORD (string|path)> \
+    -ethController <CONTROLLER_CONTRACT_ADDRESS (required if '-network' not provided)>
+```
+
+## Single Orchestrator with Redeemer
+
+This setup allows an Orchestrator to use a seperate Ethereum account for redeeming winning tickets on-chain and paying for that transaction while the ticket recipient is still Orcherstrator's on-chain registered Ethereum address.
+
+The Orchestrator node will still be responsible for initialising rounds and calling `reward`*.
+
+\* _currently, only the on-chain registered address can call `reward`_
+
+1. Start the Redeemer 
+
+```shell
+livepeer \
+    <...ETH SETUP...> \
+    -redeemer \ 
+    -httpAddr <REDEEMER_HTTP_ADDR (host):port> \ 
+    -ethOrchAddr <ORCHESTRATOR_ON_CHAIN_ETH_ADDR (also the recipient address)>
+```
+
+2. Start the Orchestrator
+
+
+```shell
+livepeer \
+    <...ETH SETUP...> \
+    -orchestrator -transcoder \
+    -initializeRound \
+    -httpAddr <ORCH_HTTP_ADDR (host):port \
+    -redeemerAddr <REDEEMER_HTTP_ADDR> \
+    -pricePerUnit <PRICE (wei/pixel if '-pixelsPerUnit' is not set)>
+```
+
+
+## Blockchain Service Node with (Multiple) Orchestrator(s)
+
+In this setup a node started with the keys for the on-chain registered address will be responsible for all transactions. 
+
+In order to use multiple orchestrators with this setup, the cluster of orchestrator nodes responsible for transcoding would have to be behind a load balancer (e.g. NGINX). The URI of the load balancer will be the on-chain registered Service URI.
+
+1. Start the Blockchain Service Node 
+
+```shell
+livepeer \
+    <...ETH SETUP...> \
+    -redeemer \ 
+    -httpAddr <REDEEMER_HTTP_ADDR (host):port> \
+    -initializeRound 
+```
+
+2. Start an Orchestrator 
+```shell
+livepeer \
+    <...ETH SETUP...> \
+    -orchestrator -transcoder \
+    -ethOrchAddr <ORCHESTRATOR_ON_CHAIN_ETH_ADDR> \
+    -redeemerAddr <REDEEMER_HTTP_ADDR> \
+    -httpAddr <ORCH_HTTP_ADDR (host):port> \
+    -pricePerUnit <PRICE (wei/pixel if '-pixelsPerUnit' is not set)>
+```
+
+## Redeemer + RewardService & RoundInitializer + Multiple Orchestrators
+
+This is the "coldest" setup possible for on-chain registered addresses. The keys for the on-chain registered address will be responsible for initialising rounds and calling reward.
+
+To use this setup with multiple orchestrators a load balancer is required as described above. 
+
+1. Start the Redeemer
+
+```shell
+livepeer
+    <...ETH SETUP...> \
+    -redeemer \ 
+    -httpAddr <REDEEMER_HTTP_ADDR (host):port> \ 
+    -ethOrchAddr <ORCHESTRATOR_ON_CHAIN_ETH_ADDR>
```

Comment:
With the recent changes in #1547 we can add a note that the address specified via `-ethAcctAddr` should be different than the address specified by `-ethOrchAddr` (or else the reward service will be started with this command).

Reply:
We already have a note that states the address should be that of the rewardservice, but i'll elaborate on it.

Reply:
fixed in 86cebba